aiidalab_widgets_base/export.py

```diff
@@ -40,8 +40,6 @@ def export_aiida_subgraph(self, change=None):  # pylint: disable=unused-argument
             document.body.appendChild(link);
             link.click();
             document.body.removeChild(link);
-            """.format(
-                payload=payload, filename=f"export_{self.process.id}.aiida"
-            )
+            """.format(payload=payload, filename=f"export_{self.process.id}.aiida")
```

Comment:
I think this can be a f-string

```python
f"""
var link = document.createElement('a');
link.href = "data:;base64,{payload}";
link.download = "export_{self.process.id}.aiida";
document.body.appendChild(link);
link.click();
document.body.removeChild(link);
"""
```

Reply:
It can, but the current `.format` is more readable imo. Because we're interpolating a string that represents Javascript code, it would be quite hard to see which variables and where are they inserted.